doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -77,25 +76,44 @@ The following tables present memory requirements for samples running on the :ref
 
 .. table:: nRF52833 single protocol memory requirements
 
-   +------------------------------------+-----------+-----------------------------------+-------------------------------+---------------------------+-----------------+-----------------------------------+-------------------------------+-----------------+
-   | Sample                             | Variant   |   ROM OpenThread stack + App [kB] |   ROM Bluetooth LE stack [kB] |   Persistent storage [kB] |   Free ROM [kB] |   RAM OpenThread stack + App [kB] |   RAM Bluetooth LE stack [kB] |   Free RAM [kB] |
-   +====================================+===========+===================================+===============================+===========================+=================+===================================+===============================+=================+
-   | :ref:`CLI <ot_cli_sample_minimal>` | master    |                               375 |                             0 |                        24 |             113 |                                88 |                             0 |              40 |
-   +------------------------------------+-----------+-----------------------------------+-------------------------------+---------------------------+-----------------+-----------------------------------+-------------------------------+-----------------+
-   | :ref:`CLI <ot_cli_sample_minimal>` | ftd       |                               356 |                             0 |                        24 |             132 |                                87 |                             0 |              41 |
-   +------------------------------------+-----------+-----------------------------------+-------------------------------+---------------------------+-----------------+-----------------------------------+-------------------------------+-----------------+
-   | :ref:`CLI <ot_cli_sample_minimal>` | mtd       |                               298 |                             0 |                        24 |             190 |                                79 |                             0 |              49 |
-   +------------------------------------+-----------+-----------------------------------+-------------------------------+---------------------------+-----------------+-----------------------------------+-------------------------------+-----------------+
+   +------------------------------------+-----------+---------------------------+----------------------+---------------------------+-----------------+---------------------------+----------------------+-----------------+
+   | Sample                             | Variant   |   ROM OT stack + App [kB] |   ROM BLE stack [kB] |   Persistent storage [kB] |   Free ROM [kB] |   RAM OT stack + App [kB] |   RAM BLE stack [kB] |   Free RAM [kB] |
```

Comment:
We cannot use BLE in the docs.
You could try forcing line breaks in the column headers with a line block - I know this works in list tables, not sure if you can do it in ascii tables as well ...
In list tables you would use it like this:
```
   * - Line-break example
     - x
     - | First line
       | Second line
```

Reply:
The table is generated automatically by `python-tabulate`. By the moment I'm okay with going back to Bluetooth LE.